scripts/distributables/osx-build.md

```diff
@@ -48,10 +48,10 @@ EOF
     ./configure
     make -j5
     make binary-dist
-    #ghc is built now in a file named ghc-9.0.2-x86_64-apple-darwin.tar.xz
+    #ghc is built now in a file named ghc-9.2.5-x86_64-apple-darwin.tar.xz
     
     # install this ghc using ghcup
-    ghcup install ghc -u "file:///$HOME/ghc/ghc-9.0.2-x86_64-apple-darwin.tar.xz" ghc-simple
+    ghcup install ghc -u "file:///$HOME/ghc/ghc-9.2.5-x86_64-apple-darwin.tar.xz" ghc-simple
```

Comment:
I think in principle you also want to "bump" the line below. I.e. `8.10.7`. -> `9.2.5`

However I do not recall whether this custom ghc is used? So maybe this should just be removed? @abizjak

Reply:
This entire file is obsolete. The concordium-client is no longer a static build on mac. We build a package with shared libraries since mid this year.

The instructions are in distributables/macOS-package/README.md

Reply:
Should it be removed then?

Reply:
Yes.